app/assets/stylesheets/spree/backend/solidus_reviews.css

```diff
@@ -1,3 +1,4 @@
 /*
+ *= require jquery-bar-rating
```

Comment:
I don't see a necessity for including that stylesheet in the the admin. We don't display rating stars in the backend.